src/iotjs_module_gpioctl.cpp

```diff
@@ -23,6 +25,57 @@ namespace iotjs {
 
 //-----------------------------------------------------------------------------
 
+GpioCbWrap::GpioCbWrap(JObject& jcallback, gpio_cb_t* cb)
```

Comment:
Looks like this implementation duplicate with it in "src/iotjs_gpiocbwrap.cpp" ?